src/main/java/com/stripe/model/Charge.java

```diff
@@ -79,19 +78,53 @@ public void setAmountRefunded(Long amountRefunded) {
 	}
 
 	public String getApplicationFee() {
```

Comment:
Should we annotate these with deprecated? I could see the argument either way: leaving them in may be convenient for some while taking them out may reduce the cognitive complexity of the library's API.